recipes/orographic-precipitation/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "orographic_precipitation" %}
+{% set version = "0.1rc0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "13559a0f324f9761ee2534430feb628f71416ff1c3b10c79d3452b2c276246b2"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - numpy
```

should you add ``matplotlib`` as well here?
```suggestion
    - numpy
    - matplotlib
```

---

That's fine. I included some plotting function at first but that is not really part of the package but rather helps visualising the example in my README. So I excluded the function.